fraudnet/src/__test__/index.test.ts

```diff
@@ -0,0 +1,85 @@
+import { createFraudnet } from "../index";
+import { Fraudnet, FraudnetOptions } from "../fraudnet";
+
+describe("Fraudnet", () => {
+  let options: FraudnetOptions;
+  beforeEach(() => {
+    options = {
+      env: "sandbox",
+      clientMetadataId: "cmid123",
+      cspNonce: "cspNonce1",
+      timeout: "5s",
+      sessionId: "id123",
+      fraudnetSource: "FRAUDNET_SOURCE",
+    };
+  });
+  afterEach(() => {
+    document.body.innerHTML = "";
+  });
+  describe("createFraudnet", () => {
+    it("returns a fraudnet instance", () => {
+      const fraudnet = createFraudnet(options);
+
+      expect(fraudnet).toBeInstanceOf(Fraudnet);
+    });
+  });
+  describe("loadScript", () => {
+    it("loads a fraudnet script on the page", () => {
+      // todo
+      // shouldn't load more than once on the page
+    });
+    it("loads a config script on the page", () => {
+      const fraudnet = createFraudnet(options);
+
+      fraudnet.loadFraudnet();
+
+      const sessionId = fraudnet.sessionId;
+
+      const el = document.querySelector('[fncls][type="application/json"]');
```

Comment:
I think this will fix the ts being angry at you:

```suggestion
      const el = document.querySelector('[fncls][type="application/json"]') as HTMLScriptElement;
```